astroquery/xmatch/core.py

```diff
@@ -187,12 +187,12 @@ def _parse_text(self, text):
         """
         header = text.split("\n")[0]
         colnames = header.split(",")
-        for cn in colnames:
-            if colnames.count(cn) > 1:
-                ii = 1
-                while colnames.count(cn) > 0:
-                    colnames[colnames.index(cn)] = cn + "_{ii}".format(ii=ii)
-                    ii += 1
+        for item in colnames:
```

Comment:
I would suggest to use descriptive variable names, that is also part of the motivation for this cleanup. 
E.g. `item` --> `column` or `colname`
`ii` -> could either stay or call it `count`, but `it` is not better than `ii` at all.